configure.ac

```diff
@@ -286,8 +286,18 @@ yes_no_try "$enable_legacy_links" "no"
 enable_legacy_links=$?
 AM_CONDITIONAL([BUILD_LEGACY_LINKS], [test $enable_legacy_links -ne $DISABLED])
 
-dnl --with-* options: external software support, and custom locations
+dnl --disable-nls option 
+AM_GNU_GETTEXT([external])
+AM_GNU_GETTEXT_VERSION(0.18)
+AM_CONDITIONAL([USE_NLS], [test "x${USE_NLS}" = "xyes"])
+
+# Check for disable_nls
+AS_IF([test "x${USE_NLS}" = "xyes"],
+      [
+          AC_DEFINE([PCMK__Support_NLS],[1],[Define to 1 if USE_NLS should be enabled])
```

Comment:
Our current style guide recommends all caps for defined constants, so this would be PCMK__SUPPORT_NLS